README.md

```diff
@@ -42,6 +42,24 @@ Add `plugin:relay/recommended` or `plugin:relay/strict` in `extends`:
 }
 ```
 
+### Suppressing rules within graphql tags
+
+The following rules support suppression within graphql tags:
+
+- relay/unused-fields
+- relay/must-colocate-fragment-spreads
+
+Supported rules can be suppressed by adding `# eslint-disable-next-line relay/name-of-rule` to the preceding line:
+
+```js
+graphql`fragment foo on Page {
+  # eslint-disable-next-line relay/must-colocate-fragment-spreads
+  ...unused1
+}`
+```
+
+Note that only this form of suppression works. Inline disables don't currently work until graphql-js provides support for parsing Comment nodes in their AST.
```

Comment:
I'm not sure I understand what you mean by "inline disables". To me `eslint-disable-next-line` is a type of inline disable.

Reply:
Hmm, I was referring to the eslint suppression where the comment is on the same line as the offending ast node to be "inline". I guess I could make the readme more specific

Reply:
Ah, I get it now. Thanks for the update!